src/utils/packageUtils.ts

```diff
@@ -490,3 +496,17 @@ export function copyDir(src: string, dest: string): void {
     return entry.isDirectory() ? copyDir(srcPath, destPath) : fs.copyFileSync(srcPath, destPath);
   });
 }
+
+export function copyDescriptorProperties(
```

Comment:
I don't like this pattern (pass in some objects by reference, rely on side effects to happen)

A cleaner pattern would be for the consumer to say

```
packageDescriptorJson = copyDescriptorProperties( packageDescriptorJson, definitionFileJson) 
```

and have a function that returns a new object using spread to avoid mutation
```
const copyDescriptorProperties = ( packageDescriptorJson, definitionFileJson):packageDescriptorJson =>
{
...packageDescriptorJson,
...Object.fromEntries()
}
```

Reply:
Will check